app/Tools/LumenSession.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Ushahidi Lumen Session
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\Tools;
```

Comment:
Namespacing is probably wrong here. I don't really know what to call the `Session` layer.